Lib/test/test_json/test_attrdict.py

```diff
@@ -133,7 +133,7 @@ def test_pickle(self):
         cached_module = sys.modules.get('json')
         sys.modules['json'] = self.json
         try:
-            for protocol in range(6):
+            for protocol in range(pickle.HIGHEST_PROTOCOL + 1):
```

Comment:
This change is not related, but I think it is minor enough to be included. It just replaces a magic number `6` with more context.